site/docs/v1/tech/guides/account-management.adoc

```diff
@@ -0,0 +1,190 @@
+---
+layout: doc
+title: Account Management
+description: Managing Accounts
+---
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Overview
+
+This allows you the manage accounts.
+
+* <<Setup MFA as Admin (from UI)>>
+    ** <<On Tenant Level>>
+    ** <<On Application Level>>
+* <<Setup MFA Through Self Service Page>>
+    ** <<Account Home>>
+    ** <<Troubleshooting>>
+    ** <<Add an Authentication Method - Add TOTP>>
+    ** <<Add an Authentication Method - Add SMS>>
+    ** <<Add an Authentication Method - Add Email>>
+* <<An Example>>
+* <<Add A Theme >>
```

Comment:
Extra space after `Theme` - not sure if that affects the render of the link? Perhaps not?

Reply:
Removed this in the latest